### DISCUSSION TITLE
other currencies for a columns

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [saulpw/visidata#1286] [Q&A]
Hello,
As you know I am fan of visidata.
try to work more with it.
2 questions though.
1. How can we change the currency to Euro?
2. I make a new column with a = sign, but when I hit "q" to go to last page, it will be deleted. how can I preserve it in the main sheet?

**Comment by saulpw** [comment]
Hi @Shahin-rmz,

1. The `currency` type is currently either misnamed or incomplete; as it does not track a currency marker and in fact has little to do with currency.  It is simply a "dirty float", a float that is parsed without any non-float characters--so currency symbols, thousands separators, anything that's not part of the number, are all effectively just ignored. But I can see how the column could have a "currency unit" attribute, and each currency unit would know how it should be displayed (with a format string, probably).  That would be pretty sweet.
2. Columns are added only to the current sheet.  @frosencrantz requested a command to hoist a column onto the source sheet.  I also wonder about an option that would automatically copy new derived columns on subsheets, to all parent sheets of the same type.  That also seems like it would be pretty sweet.

**Reply by Shahin-rmz** [reply]
Thanks @saulpw for the reply.
So for now, I would select my specific rows with Regex and press `n` or `N` to "focus" on them and do my calculations on them on the same sheet.
Would you please tell me what is best way to do the following scenario?

- I have a column with Kilograms and ounces units, and want to convert them into the grams.
can I do `=` with *only* selected rows, make a new row for ounces and also new row for Kilograms and merge them.
Or there is better strategy?
Best

**Reply by saulpw** [reply]
You can use `g=` to set current column in only selected rows to the given expression.
